roofit/xroofit/inc/RooFit/xRooFit/xRooFit.h

```diff
@@ -0,0 +1,181 @@
+#pragma once
+// removed the include guard because was preventing having local version of xRooFit on top of ROOT's version
+//#ifndef XROOFIT_XROOFIT_H
+//#define XROOFIT_XROOFIT_H
```

Comment:
We don't use `pragma once` in ROOT, and I would be weary of introducing a "new" pragma so short before the next ROOT release.

The header guard collision won't be a problem anyway if we move the `xRoo*.h` headers except for the `xRooBrowser.h` into the `src` directory. So please use header guards instead of the pragma.